jobs/resource_job.go

```diff
@@ -120,6 +132,10 @@ type JobSettings struct {
 	Format string            `json:"format,omitempty" tf:"computed"`
 	// END Jobs API 2.1
 
+	// BEGIN Jobs + Repo integration preview
+	GitSource *GitSource `json:"git_source,omitempty" tf:"group:git_source"`
```

Comment:
tf:group not needed here - it's meant for "at least one of these fields" groups.